### PR TITLE
Added sln

### PR DESCRIPTION
**Pull request by laska-sl**

### CODE REVIEW

**Review by sponkrashin — APPROVED** [review]
Looks good!

**Review by sponkrashin — CHANGES_REQUESTED** [review]
Hold on

File: arcadia-parties.sln [hunk 1]
```diff
@@ -0,0 +1,34 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15
+VisualStudioVersion = 15.0.26124.0
```
Comment (sponkrashin) [inline arcadia-parties.sln]: There is a strange version here. All my projects created in MSVS2019 use VisualStudioVersion = 16.X here...

**Review by sponkrashin — CHANGES_REQUESTED** [review]

File: arcadia-parties.sln (hunk 1 above)
Comment (laska-sl) [reply arcadia-parties.sln]: I've fixed this.
Removed lines

**Review by sponkrashin — APPROVED** [review]

**Pull request merged**